packages/edit-site/src/components/global-styles/shadows-panel.js

```diff
@@ -138,7 +138,7 @@ function ShadowItem( { shadow, category } ) {
 		>
 			<HStack>
 				<FlexItem>{ shadow.name }</FlexItem>
-				<FlexItem display="flex">
+				<FlexItem>
```

Comment:
The `FlexItem` needs `display="flex"`; otherwise, the icon isn't correctly aligned. I've not tested it, but the same is probably true for the font sizes component.

## Screenshots
| without flex | with flex |
|--------|--------|
| ![CleanShot 2024-12-12 at 10 31 19](https://github.com/user-attachments/assets/c9d29b43-3228-4f49-a676-009d8451438a) | ![CleanShot 2024-12-12 at 10 32 13](https://github.com/user-attachments/assets/350ef62f-725f-490b-802b-fd19bb256794) |

Reply:
Thanks for the suggestion. Implemented in the latest commit 🚀.